lib/gscraper_web/templates/registration/new.html.eex

```diff
@@ -0,0 +1,27 @@
+<div class="card card-auth">
+  <div class="card-auth__title">
+    <h4><%= dgettext "auth", "Sign up" %></h4>
+  </div>
+  <form class="form form-auth">
+    <div class="form__group required">
+      <label for="username" class="required">Username</label>
+      <input class="form__control" required="required" autofocus="autofocus" type="text" name="email">
+      <div class="form__feedback"></div>
+    </div>
+    <div class="form__group required">
+      <label for="password" class="required">Password</label>
+      <input class="form__control" required="required" autofocus="autofocus" type="password" name="password">
+      <div class="form__feedback"></div>
+    </div>
+    <div class="form__group required">
+      <label for="password_confirm" class="required">Password confirmation</label>
```

Comment:
I see you use `dgettext` for the button and title. Should we use for the label as well?

Reply:
I didn't localize these labels because these form fields will be generated using the form helpers in the backend PR in which the localizations were added in 0b23901.

Reply:
Got it 👍